docs/compatibility-chart.en.md

```diff
@@ -13,6 +13,7 @@ Carrier not being listed on the table does not mean that this patch won't work f
 | Asia          | Republic of Korea (450) | SKT (05)          | △                   | Not operational                         | OMD Registration with `OMD 기타LTE핸드셋_VOLTE` or `OMD Default5G` Code, IMS might be Enabled, but could be Unstable                                                                                  |
 | Asia          | Republic of Korea (450) | LG U+ (06)        | O                  | Not operational                         | OMD Registration must be Disabled                                                                                                                                                                |
 | Asia          | Republic of Korea (450) | KT (08)           | △       | Not operational                         | For only Pixel 6 / 6 Pro / 6A, OMD Registration with `PTA_VOLTE` Code, Incoming Call is Fullt Working. For Outgoing Call, to KT or Landline is Available, but to SKT or LG U+ has Noise Problem. |
+| Asia          | Israel                  | Hot Mobile (03)   | ?                  | O                                       |  Tested on Pixel 7       |
```

Comment:
Please add URL of this PR as a reference, just like other descriptions do.
 ```suggestion
| Asia          | Israel                  | Hot Mobile (03)   | ?                  | O                                       |  Tested on Pixel 7, [Reference](https://github.com/kyujin-cho/pixel-volte-patch/pull/76/)       |
```

Reply:
> Please add URL of this PR as a reference, just like other descriptions do.
>  ```suggestion
> | Asia          | Israel                  | Hot Mobile (03)   | ?                  | O                                       |  Tested on Pixel 7, [Reference](https://github.com/kyujin-cho/pixel-volte-patch/pull/76/)       |
> ```

Done.